templates/init/dataconnect/connector.yaml

```diff
@@ -1,2 +1,13 @@
 connectorId: "__connectorId__"
 authMode: "PUBLIC" 
+## ## Here's an example of how to add generated SDKs.
+## ## You'll need to replace the outputDirs with ones pointing to where you want the generated code in your app.
+# generate:
+#   javascriptSdk:
+#     outputDir: <Path where you want the generated SDK to be written to, relative to this file>
+#     package: "@firebasegen/my-connector"
+#     packageJSONDir: < Optional. Path to your Javascript app's package.json>
+#   swiftSdk:
+#     outputDir: <Path where you want the generated SDK to be written to, relative to this file>
+#   kotlinSdk:
+#     outputDir: <Path where you want the generated SDK to be written to, relative to this file>
```

Comment:
nit: add "package" under "kotlinSdk" to specify the kotlin package to incorporate into the generated kotlin code.